v3/src/components/case-table/attribute-menu.tsx

```diff
@@ -0,0 +1,156 @@
+import { MenuItem, MenuList, useToast } from "@chakra-ui/react"
+import React, { forwardRef } from "react"
+import { CalculatedColumn } from "react-data-grid"
+import { TRow } from "./case-table-types"
+
+interface IProps {
+  disableToolTips?: (show: boolean) => void;
+  column: CalculatedColumn<TRow, unknown>;
```

Comment:
```suggestion
  disableToolTips?: (show: boolean) => void
  column: CalculatedColumn<TRow, unknown>
```
Semicolons aren't required here, either.